app/models/optimized_image.rb

```diff
@@ -87,7 +87,7 @@ def self.create_for(upload, width, height, opts = {})
         end
 
         if resized
-          thumbnail = OptimizedImage.create!(
+          thumbnail = OptimizedImage.new(
```

Comment:
`create!` should be equivalent to `new..save!`

Reply:
Good catch, that was leftover from a pre-refactor where I needed to change attributes.